build.sbt

```diff
@@ -41,6 +43,8 @@ libraryDependencies  ++=  Seq(
   "com.github.alanverbner"          %% "bip39"                     % "0.1",
   "com.github.scopt"                %% "scopt"                     % "4.0.0-RC2",
   "io.scalaland"                    %% "chimney"                   % "0.3.0",
+  "com.github.julien-truffaut"      %% "monocle-core"              % monocleVersion,
+  "com.github.julien-truffaut"      %% "monocle-macro"             % monocleVersion,
```

Comment:
It might worth considering, either for now or in the future, to add monocle-law, and to consider property-based testing in our codebase in general.

Reply:
yes, sounds like a good idea